x-pack/plugins/cases/public/components/case_view/components/suggest_users_popover.tsx

```diff
@@ -173,7 +173,7 @@ const SuggestUsersPopoverComponent: React.FC<SuggestUsersPopoverProps> = ({
         searchPlaceholder: i18n.SEARCH_USERS,
         clearButtonLabel: i18n.REMOVE_ASSIGNEES,
         emptyMessage: <EmptyMessage />,
-        noMatchesMessage: <NoMatches />,
+        noMatchesMessage: searchResultProfiles ? <NoMatches /> : <EmptyMessage />,
```

Comment:
@cnasikas I don't understand why, but as soon as I type something if the results passed to `options` are `undefined` or `null` it defaults to showing the `noMatchesMessage`. Maybe that's how it's supposed to work where it only shows `emptyMessage` if you haven't typed something in the search box? This is how I was able to get the no results flickering to stop.

Even with this and the `isFetching` change it doesn't show the loading state for very long. I also tried just setting `isLoading` to `true` and adding a spinner for the `loadingMessage`. I don't see the `loadingMessage` being displayed and the only thing that changes is that there's a spinner to the right of the searchbox.

Reply:
Hey! Let's merge this PR and resolve the issue on another PR.